Controllers/Frontend/PaypalUnifiedApm.php

```diff
@@ -180,4 +221,32 @@ public function returnAction()
             'requireContactToMerchant' => true,
         ]);
     }
+
+    /**
+     * @return string
+     */
+    private function getCaptureId(Order $payPalOrder)
```

Comment:
please use `\SwagPaymentPayPalUnified\Components\Services\OrderPropertyHelper`

Reply:
Nice... i searched for that but i only found the method in the AbstractPaypalPaymentController